badCode.rb

```diff
@@ -0,0 +1,7 @@
+def badName
+  if something
+    test
+    end
+sum=1+2
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Layout/SpaceAroundOperators: Surrounding space missing for operator =.